qutebrowser/config/config.py

```diff
@@ -523,6 +525,13 @@ def set_command(self, win_id: {'special': 'win_id'},
                 val = self.get(sectname, optname[:-1], transformed=False)
                 message.info(win_id, "{} {} = {}".format(
                     sectname, optname[:-1], val), immediately=True)
+            elif optname.endswith('!'):
+                val = self.get(sectname, optname[:-1])
+                layer = 'temp' if temp else 'conf'
+                if type(val) == type(True):
+                    self.set(layer, sectname, optname[:-1], str(not val))
+                else:
+                    raise cmdexc.CommandError("set: Attempted inversion of non-boolean value. Aborting.")
```

Comment:
I'd prefer to remove the *Aborting.* here. Every error in the statusbar implies something being aborted.